libraries/ashrae/guideline36/components.yml

```diff
@@ -58,3 +60,26 @@ heating-coil:
   - template: https://brickschema.org/schema/Brick#Position_Command
     library: https://brickschema.org/schema/1.3/Brick
     args: {"name": "cmd"}
+
+cooling-coil:
+  body: >
+    @prefix p: <urn:___param___#> .
+    @prefix brick: <https://brickschema.org/schema/Brick#> .
+    p:name a brick:Cooling_Coil ;
+      brick:hasPoint p:cooling_signal .
+  dependencies:
+    - template: https://brickschema.org/schema/Brick#Cooling_Command
+      library: https://brickschema.org/schema/1.3/Brick
+      args: {"name": "cmd"}
+
+filter:
+  body: >
+    @prefix p: <urn:___param___#> .
+    @prefix brick: <https://brickschema.org/schema/Brick#> .
+    p:name a brick:Filter ;
+      brick:hasPoint p:pressure_drop .
+  dependencies:
+    - template: https://brickschema.org/schema/Brick#Filter_Differential_Pressure_Sensor
+      library: https://brickschema.org/schema/1.3/Brick
+      args: {"name": "filter_pd"}
```

Comment:
`filter_pd` should appear somewhere in the body, or you should change `filter_pd` here to `pressure_drop`